src/wx/cmdevents.cpp

```diff
@@ -2766,10 +2769,26 @@ EVT_HANDLER_MASK(ChangeFilter, "Change Pixel Filter", CMDEN_NREC_ANY)
     }
 
     wxString msg;
-    msg.Printf(_("Using pixel filter %s"), pixel_filters->GetString(gopts.filter));
+    msg.Printf(_("Using pixel filter:") + " %s", pixel_filters->GetString(gopts.filter));
     systemScreenMessage(msg);
 }
 
+static const wxString&
+ifb_short_name(const wxChoice* ifbs, unsigned num)
+{
+    static std::map<unsigned, wxString> cache;
+
+    if (cache.count(num))
+        return cache[num];
+
+    // Shorten and clean up name.
+    cache[num] = strutils::trim(strutils::titlecase(
+        strutils::remove_word(strutils::remove_word(ifbs->GetString(num).Lower(), _("interframe")), _("blending"))
```

Comment:
This is going to be confusing for localizers. Without context, it is not clear what these 2 strings are referring to and that they need to be identical to how "interframe blending" was translated elsewhere. In addition, not every language will translate that expression in 2 words, some may use 3 or more.
The case for the remaining word is also different than in its original context, which has different inflections in some languages.
Finally, not every language uses spaces for word delimiters.

Instead, I would suggest hardcoding the strings we want here. This is not duplication in the sense that the difference of context means they are different localizable strings even though they happen to be the same in English.